src/Movie-Card/Movie-Card.js

```diff
@@ -1,7 +1,26 @@
 import React from 'react'
+import './Movie-Card.css'
+const MovieCard = ({ movieDetails, switchView}) => {
+    const {
+        title, poster_path, id, release_date, average_rating
+    } = movieDetails;
 
-const MovieCard = ({ id, title, poster, switchView }) => {
-    return <img key={id} src={poster} id={id} name={title} alt={title} className='movie-card' height='300' width='200' onClick={() => switchView(id)}/>
-}
+    const releaseDate = new Date(release_date).toLocaleDateString('en-US')
+    const avgRating = average_rating.toFixed(1)
+    return <article  key={id} id={id} name={title} alt={title} className='movie-card' onClick={() => switchView(id)}>
```

Comment:
I made Movie-Card an article instead of a div, let me know if you think this should be something else or layed out differently!

Reply:
I think article works great!